desktop/src/main/kotlin/com/github/picture2pc/desktop/ui/main/elements/RotationButtons.kt

```diff
@@ -35,6 +38,7 @@ fun RotationButtons(
         ) {
             mDVM.rotationState.value =
```

Comment:
Not good to do here, if needed can be done in pDVM because it already has the mDVM